### DISCUSSION TITLE
Downloading Mission Specific Data

### DISCUSSION THREAD

**Discussion by pruthviacharya** [DOI-USGS/ISIS3#5217] [Q&A] [answered]
This is my first time installing ISIS and I was able to successfully install ISIS version 4.3.0 on Ubuntu 22.04.2 LTS using Anaconda. 

I am currently trying to get ancillary data for the Mars Reconnaissance Orbiter using 
`rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isisdata/data/mro .`, based on the [GitHub installation guide](https://github.com/DOI-USGS/ISIS3/tree/4.3). 

However, I am getting an error `rsync: [Receiver] failed to connect to isisdist.astrogeology.usgs.gov (184.164.152.106): Connection timed out (110)
rsync error: error in socket IO (code 10) at clientserver.c(139) [Receiver=3.2.7]`. 

Is this is known issue? And is there a fix for this?

Thanks

**Comment by pruthviacharya** [comment, marked as answer]
A workaround I found was the clone the GitHub repository and then run `isis/scripts/downloadIsisData`

**Reply by Kelvinrr** [reply]
yeah those are the old docs you were looking at. I would make an issue to backport that script.
